psi/pmtdescriptor.go

```diff
@@ -49,6 +49,14 @@ const (
 	EC3                uint8 = 204 // 1100 1100 (0xCC)
 )
 
+// Audio service type
```

Comment:
Could you mention it is the `Iso639 Audio Type` in the comment? Sorry to be picky. :O

Reply:
Sure